packages/mobile/src/tokens/saga.test.ts

```diff
@@ -54,3 +56,33 @@ describe(importTokenInfo, () => {
       .run()
   })
 })
+
+describe(tokenAmountInWei, () => {
+  const mockAddress = '0xMockAddress'
+
+  it('map to token amount successfully', async () => {
+    await expectSaga(tokenAmountInWei, new BigNumber(10), mockAddress)
+      .withState(
+        createMockStore({
+          tokens: {
+            tokenBalances: {
+              [mockAddress]: {
+                address: mockAddress,
+                decimals: 5,
+              },
+            },
+          },
+        }).getState()
+      )
+      .returns((10e5).toFixed(0))
```

Comment:
nit: usually better to avoid intermediary representations in expectations
```suggestion
      .returns('1000000')
```